service/tutorial/order/NewTutorialServices.xml

```diff
@@ -0,0 +1,322 @@
+                    <set field="unitAmount" from="orderItem.unitAmount"/>
+                    <set field="orderMap"
+                         from="[orderId:orderItem.orderId,currencyUomId:orderItem.currencyUomId,
+                         itemDescription:orderItem.itemDescription,productId:orderItem.productId,quantity:orderItem.quantity,
+                         unitAmount:orderItem.unitAmount]"/>
+                    <set field="order" from="order+orderMap"/>
+                    <service-call name="mantle.order.OrderPart.create#OrderItem" in-map="context+[orderId:orderId]"/>
+                </if>
+            </iterate>
+        </actions>
+    </service>
+
+    <!--Get all Orders-->
+
+    <service verb="get" noun="OrderList">
+        <out-parameters>
+            <parameter name="orders" type="list">
+                <parameter name="orderId"/>
+                <parameter name="orderName"/>
+                <parameter name="currencyUomId"/>
+                <parameter name="salesChannelEnumId"/>
+                <parameter name="statusId"/>
+                <parameter name="placedDate"/>
+                <parameter name="grandTotal"/>
+                <parameter name="customer_details" type="map">
+                    <parameter name="firstName"/>
+                    <parameter name="middleName"/>
+                    <parameter name="lastName"/>
+                </parameter>
+                <parameter name="order_parts" type="list">
+                    <parameter name="orderPartSeqId"/>
+                    <parameter name="partName"/>
+                    <parameter name="facilityId"/>
+                    <parameter name="shipmentMethodEnumId"/>
+                    <parameter name="partStatusId"/>
+                    <parameter name="partTotal"/>
+                    <parameter name="item_details" type="list">
+                        <parameter name="orderItemSeqId"/>
+                        <parameter name="productId"/>
+                        <parameter name="itemDescription"/>
+                        <parameter name="quantity"/>
+                        <parameter name="unitAmount"/>
+                    </parameter>
+                </parameter>
+            </parameter>
+        </out-parameters>
+        <actions>
+            <set field="orders" from="[]"/>
+            <entity-find entity-name="mantle.order.OrderHeader" list="orderList">
+                <select-field field-name="orderId"/>
+                <select-field field-name="orderName"/>
+                <select-field field-name="currencyUomId"/>
+                <select-field field-name="salesChannelEnumId"/>
+                <select-field field-name="statusId"/>
+                <select-field field-name="placedDate"/>
+                <select-field field-name="grandTotal"/>
+            </entity-find>
+            <set field="customer_details" from="[]"/>
+            <entity-find entity-name="mantle.party.Person" list="personList">
```

Comment:
Why retrieve all persons list, we must fetch the person details from the customerPartyId in OrderPart for each order.